api/src/main/java/org/motechproject/nms/api/web/KilkariController.java

```diff
@@ -88,7 +88,7 @@
     public InboxResponse getInboxDetails(@RequestParam(required = false) Long callingNumber,
                                          @RequestParam(required = false) Long callId) {
 
-        log("/kilkari/inbox", String.format("callingNumber=%s, callId=%s",
+        log("REQUEST: /kilkari/inbox", String.format("callingNumber=%s, callId=%s",
```

Comment:
One method had that in it's description I removed it and decided to leave
GET of as the default for no good reason.

On Thu, Aug 6, 2015 at 8:16 AM, Frank Huster <notifications@github.com>
wrote:

> In api/src/main/java/org/motechproject/nms/api/web/KilkariController.java
> <https://github.com/motech-implementations/mim/pull/557#discussion_r36426475>
> :
>
> > @@ -88,7 +88,7 @@
> >      public InboxResponse getInboxDetails(@RequestParam(required = false) Long callingNumber,
> >                                           @RequestParam(required = false) Long callId) {
> >
> > -        log("/kilkari/inbox", String.format("callingNumber=%s, callId=%s",
> > +        log("REQUEST: /kilkari/inbox", String.format("callingNumber=%s, callId=%s",
>
> If we're marking posts with (POST), should we also mark gets with (GET)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/557/files#r36426475>.
>